programs/pluto/ikev2_parent.c

```diff
@@ -4215,7 +4228,82 @@ static bool ikev2_rekey_child_resp(struct ike_sa *ike, struct child_sa *child,
 	return true;
 }
 
-static bool ikev2_rekey_child_copy_ts(struct child_sa *child)
+/**
+ * ikev2_rekey_child_copy_sec_ctx: Copy the security label from the state of
+ * the predecessor child/IPsec SA to the state of the new child/IPsec SA
+ * replacing said predecessor during rekeying.
+ *
+ * @param[in,out]	new_state	State of the new child/IPsec SA.
+ * @param[in]		predecessor	State of the predecessor child/IPsec SA.
+ *
+ * @return	True if copying the security label from `predecessor` to
+ * 		`new_state` was successful.
+ */
+static bool ikev2_rekey_child_copy_sec_ctx(struct state *const new_state,
+					   struct state const* const predecessor) {
+	libreswan_log("%s: Prior to updating new SA's security label: predecessor security label = %s, new child/IPsec SA security label = %s",
```

Comment:
would be nice to split this log line into to lines.

Reply:
Fixed.